src/serialization.jl

```diff
@@ -0,0 +1,34 @@
+struct Save
+    filename::String
+    method
+end
+
+Save(filename="machine.jlso"; method=serialize) =
+    Save(filename, method)
+
```

Comment:
This doesn't quite look right. I want to be able to do `Save(filename="machine.jlso")` but that doesn't work. 

For consistency with existing controls, I think all the following should work:

`Save("machine.jlso")`
`Save("machine.jslo", method=...)`
`Save(filename="machine.jlso")`
`Save(filename="machine.jlso", method=...)`